### ISSUE TITLE
[PD] Developers daily activities

### ISSUE THREAD

**Issue by elmira-coding** [elmira-coding/Coursework-Planner#59]
From Induction created by [kfklein15](https://github.com/kfklein15): CodeYourFuture/Induction#3  

### Coursework content

Research a developer's daily activities and behaviours. What will you expect to know or do when you get your first job?  Activities are specific actions, tasks, or operations to accomplish a particular goal or objective.  For example: A marketing analyst must conduct market research and data analysis to identify consumer trends Behaviour is the observable actions, reactions, or conduct of someone.  For example: A marketing analyst must be able to effectively communicate complex data clearly and understandably to stakeholders.

### Estimated time in hours

1

### What is the purpose of this assignment?

- [ ]  Research a developer's daily activities and behaviours
- [ ]  Write at least 50 words on a Google Docs file
- [ ]  Check the grammar using an available too, for example, Grammarly
- [ ]  Share the file with your cohort on the "Developers daily activities" thread. If the thread doesn't exist yet, create it.
- [ ]  Read two contents shared with you
- [ ]  Leave at least 2 comments or questions on each text you read

### How to submit

Add the link to your Google Docs
Add a screenshot of your post sharing your coursework

**Comment by kfklein15** [comment]
@elmira-coding , this is in review, but I cannot see the link to the doc. Could you add it and let me know?

**Comment by elmira-coding** [comment]
Hello,
I don't know what I should add!
Could you clarify me?Thanks

**Comment by kfklein15** [comment]
Of course. In the "How to submit" session we describe what evidence needs to be sent. In this case, it is:
- Add the link to your Google Docs
- Add a screenshot of your post sharing your coursework

**Comment by kroker42** [comment]
Hi, how did you get on with sharing the document link? Do you need any more help with this?
